nucliadb/nucliadb/search/api/v1/find.py

```diff
@@ -191,6 +195,7 @@ async def find(
     x_nucliadb_user: str,
     x_forwarded_for: str,
     do_audit: bool = True,
+    raise_on_incomplete_results: bool = False,
```

Comment:
this feels awkward -- to cause an exception to be raised from a parameter.

Reply:
you are right. let me try to refactor